policyengine_us/variables/gov/states/hi/tax/income/exemptions/hi_exemptions.py

```diff
@@ -0,0 +1,44 @@
+from policyengine_us.model_api import *
+
+
+class hi_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii exemptions"
+    unit = USD
+    documentation = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=20"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.HI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.exemptions
+        person = tax_unit.members
+
+        num_exemp = tax_unit("exemptions", period)
+        is_disabled = person("is_disabled", period)
+        is_spouse = person("is_tax_unit_spouse", period)
+        spouse_disabled = tax_unit("spouse_is_disabled", period)
+        spouse_aged = tax_unit("aged_spouse", period)
+
+        return where(
```

Comment:
make the structure: 
base_head = ...
base_spouse = ...
head_disabled = where(head_is_disabled, ....) 
spouse_disabled = where(spouse_is_disabled, ....) 
return x + y+ z + a